Boards.h

```diff
@@ -603,7 +603,7 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define IS_PIN_SERVO(p)         ((p) >= 0 && (p) < MAX_SERVOS)
 #define IS_PIN_I2C(p)           ((p) == 16 || (p) == 17)
 #define IS_PIN_SPI(p)           ((p) == SS || (p) == MOSI || (p) == MISO || (p) == SCK)
-#define IS_PIN_SERIAL(p)        ((p) == 8 || (p) == 9 || (p) == 10 || (p) == 11)
+#define IS_PIN_SERIAL(p)        ((p) == 8 || (p) == 9
```

Comment:
Missing a closing paren. Please squash the commits after adding the paren if you're comfortable doing that.